src/sass/pages/_search.scss

```diff
@@ -0,0 +1,62 @@
+@use "sass:map";
+@use "../abstracts" as abstracts;
+
+.filter-wrapper {
+  background-color: abstracts.$primary-grey;
+  min-height: 100%;
+  width: 300px;
```

Comment:
Tämä width aiheuttaa ongelmia kapealla ruudulla, kun sivubaari siirtyy sivun ylimmäiseksi. Helpoin ratkaisu olisi varmaan tehdä kapealle screenille oma leveys `@media only screen and (max-width: abstracts.$medium-width)`llä ja sen sisälle sitten .filter-wrapperille width 100%.